api/client/src/main/java/org/projectnessie/client/auth/oauth2/MustFetchNewTokensException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.auth.oauth2;
+
+class MustFetchNewTokensException extends RuntimeException {
+  public MustFetchNewTokensException(String message) {
```

Comment:
Since the class is not `public` we should probably keep the constructor non-public too.